deps/v8/src/libplatform/task-queue.cc

```diff
@@ -40,11 +44,22 @@ Task* TaskQueue::GetNext() {
         process_queue_semaphore_.Signal();
         return NULL;
       }
+
     }
     process_queue_semaphore_.Wait();
   }
 }
 
+void TaskQueue::PurgeWorkers() {
+  base::LockGuard<base::Mutex> guard(&lock_);
```

Comment:
Do we want to add `DCHECK(!terminated_)` here and in `StopPurgeWorkers()` below? Entirely optional and low-priority — it's just a debugging assertion — but it would be consistent with the rest of this file.